WcaOnRails/app/webpacker/components/Panel/Wfc/DuesRedirect.jsx

```diff
@@ -115,6 +124,21 @@ export default function DuesRedirect() {
           </Form>
         </Modal.Content>
       </Modal>
+      <Confirm
+        open={toDeleteId}
+        content="Are you sure you want to delete?"
+        onCancel={() => setOpen(false)}
```

Comment:
Should this set the id to null?

Reply:
Ya it was supposed to be `setToDeleteId(null)`, I missed it. Thanks.